src/MCPClient/lib/archivematicaClient.py

```diff
@@ -107,19 +108,16 @@ def get_supported_modules(file_):
 @auto_close_db
 def handle_batch_task(gearman_job, supported_modules):
     module_name = supported_modules.get(gearman_job.task)
-    gearman_data = cPickle.loads(gearman_job.data)
+    gearman_data = six.moves.cPickle.loads(gearman_job.data)
```

Comment:
Didn't we have to deal with a pickle version protocol change between Py2 and 3?

Reply:
I set `protocol=0` in all the pickle `dump` calls in https://github.com/artefactual/archivematica/pull/1731/commits/ac1152745c460da10617a9de996218ad9b6c45fd